bad_connector_woocommerce/models/sale_order/importer.py

```diff
@@ -246,6 +246,26 @@ def woo_coupon(self, record):
         coupon_codes = [coupon.get("code") for coupon in woo_coupons]
         return {"woo_coupon": ", ".join(coupon_codes)}
 
+    @mapping
+    def woo_payment_mode_id(self, record):
+        """Mapping for woo_payment_mode_id"""
+        payment = self.env["woo.payment.gateway"].search(
+            [("external_id", "=", record.get("payment_method"))], limit=1
+        )
```

Comment:
either have one method for payment mode and process id or have common method for searching